### ISSUE TITLE
Reorganizing the folder

### ISSUE THREAD

**Issue by snakedog116** [justintime/SmartThings-Monoprice-11990#1]
Hello @justintime !

First of all, thanks for creating the code for the monoprice dual relay. I have 1 of these, and your code is a life-saver.

I'm just wondering if it would be possible for you to reorganize your folder structure to match Smartthings' recommended folder hierarchy. This would allow users like myself to quickly check for updates and sync them down to Smartthings from withing the Smartthings' API website.

I understand if that's not on your list of priorities, but it sure would be nice for people like myself using your code.

Again, thanks!

**Comment by justintime** [comment]
Actually, I was going to do that anyway.  If you have time to set it up and report back here if it worked or not, I'd appreciate it.  If it works, I'll update this repo to just point over here:

https://github.com/justintime/SmartThingsPublic

I just pushed it up, so it should be there now.

**Comment by snakedog116** [comment]
Beautiful.

It took a bit to make sure it was working... I have too many repos with the name "SmartThingsPublic" and because you didn't change anything in the code it wouldn't update, but I changed my local code, and was able to sync it with your new Repo. (Replacing my changes. I can't upload to your repo obviously.)

That's awesome, thanks for making this available like this.

Will you now be updating the code in both Repos? or will you be abandoning one of them?

**Comment by snakedog116** [comment]
Beautiful.

It took a bit to make sure it was working... I have too many repos with the name "SmartThingsPublic" and because you didn't change anything in the code it wouldn't update, but I changed my local code, and was able to sync it with your new Repo. (Replacing my changes. I can't upload to your repo obviously.)

That's awesome, thanks for making this available like this.

Will you now be updating the code in both Repos? or will you be abandoning one of them?

**Comment by justintime** [comment]
I'll remove the code from this repo and point everyone over to the new one.

I'd like to rename the SmartThingsPublic repo to ST-Monoprice-Dual-Relay.  It'll break your integration until you fix it, but I think it's a better approach.  I see bravenel does that with his code.

I'll re-open this, and do the rename, then you can make sure it works as intended.

**Comment by justintime** [comment]
Try this one: https://github.com/justintime/ST-Monoprice-Dual-Relay-Adapter

**Comment by snakedog116** [comment]
Looks like the smartapp works, but the device isn't. (I had some issues with the device previously too, but got it working.

I've ended up deleting the device and trying to re-import (Did you change the name of the app from module to adapter?
But even when I deleted it completely it gives an error when trying to download it.

Note, I then tried to download it from your smartthingspublic repo and got an error again.

I am able to download it from my personal repo.
